src/providers/wallet/wallet.ts

```diff
@@ -886,7 +886,7 @@ export class WalletProvider {
       // no future transactions...
       if (tx.time > now) tx.time = now;
 
-      if (tx.confirmations === 0) {
+      if (tx.confirmations === 0 && wallet.coin === 'btc') {
         const coins = await this.getCoinsForTx(wallet, tx.txid);
```

Comment:
Seems that you don't need `coins`

Reply:
I do I don’t no how to put them in a wallet